src/main/java/taxi/controller/car/GetAllCarsController.java

```diff
@@ -18,6 +18,8 @@ public class GetAllCarsController extends HttpServlet {
     public void doGet(HttpServletRequest req, HttpServletResponse resp)
             throws ServletException, IOException {
         List<Car> cars = carService.getAll();
+        String h1 = "My all cars";
+        req.setAttribute("h1", h1);
```

Comment:
Why do you need this? You have code duplication here and in GetMyCurrentCarsController
It should be on jsp